website/components/blog/Categories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import Link from "next/link"
+
+export interface Props {
```

Comment:
In general I lean toward using `type` instead of `interface` - it's easier to perform modifications on types

Reply:
ok, switched to types